telcell/celldb/__main__.py

```diff
@@ -0,0 +1,84 @@
+import sys
+from functools import partial
+from typing import Sequence
+
+import click
+from tqdm import tqdm
+
+from telcell.celldb import pgdatabase
+from telcell.utils import script_helper
+
+
+def create_antenna_light_table(con):
+    pgdatabase.create_table(con)
+    tablename = "antenna_light"
+    with con.cursor() as cur:
+        cur.execute(
+            f"""
+            INSERT INTO {tablename}(date_start, date_end, radio, mcc, mnc, lac, ci, eci, rd, azimuth)
+            SELECT start, "end",
+                CASE
+                    WHEN radio = '2G' THEN 'GSM'
+                    WHEN radio = '3G' THEN 'UMTS'
+                    WHEN radio = '4G' THEN 'LTE'
+                    WHEN radio = '5G' THEN 'NR'
+                    ELSE radio
+                END radio,
+                204, mnc,
+                lac,
+                CASE
+                    WHEN radio = '4G' THEN NULL
+                    ELSE cell
+                END ci,
+                CASE
+                    WHEN eci is NOT NULL THEN eci
+                    WHEN radio = '4G' AND cell is NOT NULL THEN cell
+                    ELSE NULL
+                END eci,
+                'SRID=4326;POINT('||rdx||' '||rdy||')', azimuth
+            FROM antenna
+            WHERE cell is NOT NULL OR eci is NOT NULL
+        """
+        )
+
+
+@click.group()
```

Comment:
zoals eerder vandaag besproken, ik ben niet zo'n fan van click hier. We gebruiken normaal eigenlijk altijd argparser, dus tenzij je bepaalde dingen niet kunt doen, zou ik toch voorstellen om het om te schrijven daarnaartoe

Reply:
Ik ben het met je eens maar zou het graag voor een nieuwe PR laten